core/templates/pages/topic-editor-page/modal-templates/questions-opportunities-select-difficulty-modal.component.html

```diff
@@ -39,10 +39,19 @@ <h3 class="oppia-modal-header" *ngIf="linkedSkillsWithDifficulty.length > 0">
             (click)="startQuestionCreation()"> Continue
     </button>
   </div>
+</div>
+
+<div>
   <div *ngIf="linkedSkillsWithDifficulty.length === 0">
```

Comment:
Looks like an extra unnecessary `div` here? (not sure!)

Reply:
That's true, the `div` might be unnecessary. My goal was to make it similar in style to the container above it which holds the continue and cancel buttons, but that might not be necessary.